src/lib.rs

```diff
@@ -138,6 +138,53 @@ impl Overlay {
     }
 }
 
+/// For when you can't give away ownership of the window.
+pub struct BorrowedOverlay {
+    active: bool,
```

Comment:
I think this should be `Option<bool>`?  

When I do 
```rust
    let mut overlay = overlay::BorrowedOverlay::new(255, 255);
    let window = gl_window.window();
    overlay.deactivate(window);
```
the window not gonna be clickthrough. because the initial state of `activate` is false, it doesn't exec deactivate().

I know I can activate then deactivate. but I think `Option<bool>` is better.

Reply:
Thanks for catching this. I've gone ahead and used `Option<bool>` rather than `bool`.